gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleJdksConfigs.java

```diff
@@ -96,7 +96,9 @@ public final void action() {
             throw new RuntimeException(
                     "No JDKs were configured for the gradle setup. Please run `./gradlew setupJdks` to generate the"
                             + " JDKs and ensure that you have configured JDKs properly for gradle-jdks as per"
-                            + " the readme: https://github.com/palantir/gradle-jdks#usage");
+                            + " the readme: https://github.com/palantir/gradle-jdks#usage"
+                            + " if you see this error after removing all `gradle-sls-docker` plugins"
```

Comment:
I'd make sure to add a full stop and capitalise the next character. I'd probably also indicate this is only relevant to palantirians. Something like ". Palantirians: If you see this error on an internal project after removing all `gradle-sls-docker` related plugins, see https://pl.ntr/2v1 to fix."